clinical-domain-agent/src/main/java/care/smith/fts/cda/TransferProcessDefinition.java

```diff
@@ -7,13 +7,16 @@
 import care.smith.fts.api.cda.CohortSelector;
 import care.smith.fts.api.cda.DataSelector;
 import care.smith.fts.api.cda.Deidentificator;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 
 public record TransferProcessDefinition(
-    String project,
-    CohortSelector cohortSelector,
-    DataSelector dataSelector,
-    Deidentificator deidentificator,
-    BundleSender bundleSender) {
+    @NotBlank String project,
+    TransferProcessConfig rawConfig,
```

Comment:
Does not checking for null yield any problems? We must expect and check for null everywhere we use it.

Reply:
Let's just check for null during construction, as for the other variables.